### ISSUE TITLE
Angular slickgrid upgrade does not working

### ISSUE THREAD

**Issue by HafsaAsif** [ghiscoding/Angular-Slickgrid#1348]
### Describe the bug

I am using angular-slickgrid 6.0.0 version in Angular 16. When I tried to upgrade angular slickgrid version from 6.0.1, my project compilation fails and give me this error
![CompileIssue](https://github.com/ghiscoding/Angular-Slickgrid/assets/12494385/8d250547-522e-4c42-a226-6481b4b40e2e)

### Reproduction

Any angular app with version : **_16_**
Angular slickgrid with version : **_6.0.0_**
just run command : **_npm install angular-slickgrid_**

It updates the package.json with version : **_6.6.6_**

Navigate to angular 16 project --> Run: **_ng serve_**
Then the above described issue will be generated.

### Environment Info

```shell
Angular CLI: 16.2.10
Node: 18.18.2
Package Manager: npm 9.8.1
OS: win32 x64


Angular: 16.2.12
... animations, cdk, common, compiler, compiler-cli, core, forms
... language-service, localize, material, platform-browser
... platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1602.10
@angular-devkit/build-angular   16.2.10
@angular-devkit/core            16.2.10
@angular-devkit/schematics      16.2.10
@angular/cli                    16.2.10
@ngtools/webpack                16.2.10
@schematics/angular             16.2.10
rxjs                            7.8.1
typescript                      5.1.6
zone.js                         0.13.3
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/Angular-Slickgrid/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [HOWTO - Quick Start](https://ghiscoding.gitbook.io/angular-slickgrid/getting-started/quick-start).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/Angular-Slickgrid/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/Angular-Slickgrid/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
It's not an Angular-Slickgrid but rather a DOMPurify bug, there's an open [issue](https://github.com/cure53/DOMPurify/issues/895) regarding this problem, they will hopefully have a fix soon. For now what you can do is set the DOMPurify version to the previous working version `3.0.6` (because 3.0.7 has the bug), you can use npm/pnpm/yarn override to set it to the previous version

For example with yarn resolution (make sure to remove the caret `^` in your override (in yarn they use `resolutions`, in npm/pnpm they use `overrides`)
```ts
// package.json
{
  resolutions: {
    "dompurify: "3.0.6"
  }
}
```

**Comment by ghiscoding** [comment]
DOMPurify pushed a new version [`3.0.8`](https://github.com/cure53/DOMPurify/releases/tag/3.0.8) which fixes the issue, my Renovate PR #1346 is also passing which confirms that the issue is fixed. So you can update without the need for the override anymore. Cheers

**Comment by HafsaAsif** [comment]
Hello, I had tried ur given solution and it is working now. Thanks a lot!
